src/epub/content.opf

```diff
@@ -101,13 +104,19 @@
 		<meta property="authority" refines="#subject-23">LCSH</meta>
 		<meta property="term" refines="#subject-23">sh2008106063</meta>
 		<meta property="authority" refines="#subject-24">LCSH</meta>
-		<meta property="term" refines="#subject-24">Unknown</meta>
+		<meta property="term" refines="#subject-24">sh00006155</meta>
+		<meta property="authority" refines="#subject-25">LCSH</meta>
+		<meta property="term" refines="#subject-25">sh85055080</meta>
+		<meta property="authority" refines="#subject-26">LCSH</meta>
+		<meta property="term" refines="#subject-26">sh2008113021</meta>
+		<meta property="authority" refines="#subject-27">LCSH</meta>
+		<meta property="term" refines="#subject-27">sh2008115694</meta>
 		<meta property="se:subject">Science Fiction</meta>
 		<meta property="se:subject">Shorts</meta>
 		<dc:description id="description">A collection of science fiction short stories by Mack Reynolds ordered by date of publication.</dc:description>
 		<meta id="long-description" property="se:long-description" refines="#description">
 			&lt;p&gt;&lt;a href="https://standardebooks.org/ebooks/mack-reynolds"&gt;Dallas McCord “Mack” Reynolds&lt;/a&gt; was an American science fiction writer who authored almost two hundred short stories and novellas, was a staple in all the major science fiction and fantasy magazines and published dozens of science fiction novels. He began his writing career in the late 1940s. His fiction focused on exploring and challenging both the socioeconomic themes of the day and the implications of the Cold War that raged throughout his career. A thoughtful writer of speculative fiction, many of Mack Reynolds’ predictions have come to pass, including the credit-card economy, remote warfare and a worldwide computer network. His thoughts about the outcomes of both the Soviet and western political and economic systems are still highly relevant.&lt;/p&gt;
-			&lt;p&gt;This collection gathers 21 of his stories that were published between 1953 and 1963 in &lt;i&gt;Analog&lt;/i&gt;, &lt;i&gt;Astounding Science Fiction&lt;/i&gt;, &lt;i&gt;Amazing Stories&lt;/i&gt; and others. Ordered by date of first publication, they range from spy adventures to the ultimate expression of corporate warfare and from a very short 1000-word story to full-blown novellas.&lt;/p&gt;
+			&lt;p&gt;This collection gathers stories that were published between 1950 and 1963 in &lt;i&gt;Analog&lt;/i&gt;, &lt;i&gt;Astounding Science Fiction&lt;/i&gt;, &lt;i&gt;Amazing Stories&lt;/i&gt; and others. Ordered by date of first publication, they range from spy adventures to the ultimate expression of corporate warfare and from a very short 1000-word story to full-blown novellas.&lt;/p&gt;
```

Comment:
Can you remove the years as well, in case we add more stories later?

Reply:
fixed